examples/material-next-app-router-ts/src/components/Theme/ThemeRegistry/theme.ts

```diff
@@ -0,0 +1,20 @@
+'use client';
+
+import { createTheme, ThemeOptions } from '@mui/material/styles';
+
+import GoogleInterFont from '@/app/fonts/fonts';
+
+// When needed::: first argument is needed if you have common enterprise theme, and second argument is to override your enterprise theme.
+// apply fonts to all other typography options like headings, subtitles, etc...
+const defaultTheme = createTheme(
+  {
+    typography: {
+      fontFamily: GoogleInterFont.style.fontFamily,
+      body1: { fontFamily: GoogleInterFont.style.fontFamily },
+      body2: { fontFamily: GoogleInterFont.style.fontFamily },
+    },
+  },
+  {} satisfies ThemeOptions,
+);
```

Comment:
```suggestion
const defaultTheme = createTheme({
  typography: {
    fontFamily: GoogleInterFont.style.fontFamily,
    body1: { fontFamily: GoogleInterFont.style.fontFamily },
    body2: { fontFamily: GoogleInterFont.style.fontFamily },
  },
});
```

Let's simplify this, we have documentation about this API if people need to take a deeper look at it.